apps/dispatch/src/assets/config.json

```diff
@@ -11,17 +11,10 @@
   "privacyPolicyVersion": 1,
```

Comment:
what about appUrl and apiUrl are they exempt from this refactoring?  what are they used for?

Reply:
These fields are used as input parameters for `biosimulations-navigation` and have not been a part of this refactor. The logic is that I am housing all links that are truly external in `config.json`.